src/main/java/service/Main.java

```diff
@@ -0,0 +1,59 @@
+package service;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.dao.ParserImpl;
+import service.dao.ParserTo;
+import service.dao.ParserToImpl;
+import service.dao.ReadFile;
+import service.dao.ReadFileImpl;
+import service.dao.WriteFile;
+import service.dao.WriteFileImpl;
+import service.storage.Balance;
+import service.storage.PerformingOperation;
+import service.storage.Purchase;
+import service.storage.Return;
+import service.storage.Supply;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final String balanceCode = "b";
+        final String supplyCode = "s";
+        final String purchaseCode = "p";
+        final String returnCode = "r";
+        // Read data from file
+        String filePath = "src/main/resources/fileFrom.csv";
+        File file = new File(filePath);
+        List<String> listFromFile = new ArrayList<>();
+        ReadFile fileDao = new ReadFileImpl();
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        // Convert data to Java object
+        ParserImpl parser = new ParserImpl(fruitTransactions);
+        fruitTransactions = parser
+                .parsedToFruitTransaction(fileDao.readFromFileToList(file,listFromFile));
```

Comment:
Pattern where we should create list to fullfill it within method is counterintuitive
Please use return value of method.
```suggestion
        ReadFile fileDao = new ReadFileImpl();
        List<String> listFromFile = fileDao.readFromFileToList(filePath)
        // Convert data to Java object
        ParserImpl parser = new ParserImpl();
        List<FruitTransaction> fruitTransactions = parser.parseFruitTransactions(listFromFile);
```

Please recheck remaining of logic to follow such schema of work

Also recheck naming through the project
Method names are verbs, interface/class names are nouns or sometimes adjectives